paasta_tools/utils.py

```diff
@@ -997,8 +998,24 @@ def get_net(self) -> str:
         """
         return self.config_dict.get("net", "bridge")
 
+    def has_bulkdata(
+        self,
+    ) -> bool:
+        return self.config_dict.get("uses_bulkdata", True)
+
     def get_volumes(self, system_volumes: Sequence[DockerVolume]) -> List[DockerVolume]:
```

Comment:
I've added a test that adding the uses_bulkdata flag adds the bulkdata volume. I've had to fix other failing tests by adding the uses_bulkdata flag set to False to prevent the bulkdata volume being added, so I think there is coverage of this method?